misk/src/main/kotlin/misk/clustering/fake/FakeCluster.kt

```diff
@@ -0,0 +1,73 @@
+package misk.clustering.fake
+
+import com.google.common.util.concurrent.Service
+import com.google.inject.Key
+import misk.DependentService
+import misk.clustering.Cluster
+import misk.clustering.ClusterHashRing
+import misk.clustering.ClusterResourceMapper
+import misk.clustering.ClusterWatch
+import misk.clustering.DefaultCluster
+import java.util.concurrent.Executor
+import java.util.concurrent.TimeUnit
+import javax.inject.Singleton
+
+/**
+ * A [FakeCluster] is a [Cluster] that is a fake. It delegates entirely to [DefaultCluster],
+ * but allows us to keep [DefaultCluster] internal and make it clear that [FakeCluster]
+ * is only intended to be used for fakes.
+ *
+ * NB(mmihic): I'd prefer for this to be in the misk-testing module, but it can't since it
```

Comment:
fwiw I'm starting to find separate testing modules not really worth it and a `Fake` or `Mock` prefix is good enough and sometimes easier to discover when in the real module

Reply:
yeah, not sure what drove the original testing module split tbh

Reply:
I’m on board with fakes in the main package. More discoverable, and fewer things to put in my .gradle files. Plus we occasionally want to use fake things in say, the development server.